cds_ils/importer/eitems/importer.py

```diff
@@ -72,9 +72,10 @@ def _eitem_has_higher_priority(self, eitem, priority=None):
         if not priority:
             priority = self.current_provider_priority
 
-        eitem_priority = current_app.config["CDS_ILS_IMPORTER_PROVIDERS"][
-            eitem_provider
-        ]["priority"]
+        PRIORITY_FOR_UNKNOWN = 100  # high number means low priority
```

Comment:
would you like to set this variable in the config? maybe we change the policy and we want to set up low prioriy, it would be nice to be able to do it via env variables